### DISCUSSION TITLE
Automated accessibility testing: How do you confirm issues?

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5400] [Accessibility]
We know automated testing is inherent in both the USWDS process ([as discussed here](https://github.com/uswds/uswds/discussions/5337)) and in other federal teams’ development plans. Many of us have had issues flagged that cannot be confirmed by other automated testing or by manual testing.

 
How does your team handle issues that are flagged as a result of an automated test but cannot be confirmed by other methods?
What methods are your go-to for confirming?

**Comment by mgifford** [comment]
I often recommend leveraging the accessibility tests from https://accessibilityinsights.io

Although not strictly aligned with the Trusted Tester methodology, it is a pretty solid approach. 

There is of course always the W3C's [WCAG-EM](https://www.w3.org/WAI/test-evaluate/conformance/wcag-em/) approach too.
